nurseconnect/templates/core/tags/topic_of_the_day.html

```diff
@@ -4,15 +4,15 @@
 {% if articles %}
     <section class="ContentList">
         <header class="ContentList-header">
-            <h1 class="ContentList-categoryLabel">{% trans "Topic of the day" %}</h1>
+            <h1 class="ContentList-categoryLabel">{% trans "Latest Articles" %}</h1>
```

Comment:
I thought they requested that we remove the heading. Did they mean it should be under "Latest Articles"?
Also, this section will only be rendered if there is a Topic of the day article. I think the heading could possibly not be rendered at all if it is here